src/patternfly/components/Label/styles.scss

```diff
@@ -0,0 +1,48 @@
+@import "../../patternfly-utilities";
+
+.pf-c-label {
+  // Component
+  --pf-c-label--PaddingTop: var(--pf-global--spacer--xs);
+  --pf-c-label--PaddingRight: var(--pf-global--spacer--sm);
+  --pf-c-label--PaddingBottom: var(--pf-global--spacer--xs);
+  --pf-c-label--PaddingLeft: var(--pf-global--spacer--sm);
+  --pf-c-label--LineHeight: var(--pf-global--LineHeight--md);
+  --pf-c-label--BorderRadius: 2px;
```

Comment:
This should use a global variable. If the design calls for 2px we should bring it back to design to confirm they are introducing another global variable.

Reply:
I designed this, it should be 3px you're right!